extensions/eclipse-che-theia-workspace/package.json

```diff
@@ -11,7 +11,8 @@
   "dependencies": {
     "@eclipse-che/api": "latest",
     "@theia/workspace": "next",
-    "@eclipse-che/theia-plugin-ext": "^0.0.1"
+    "@eclipse-che/theia-plugin-ext": "^0.0.1",
+    "yaml": "1.10.0"
```

Comment:
Don't we need a CQ for the new dependency?

Reply:
Used `js-yaml` library.